File: 1460-make-two-arrays-equal-by-reversing-subarrays/1460-make-two-arrays-equal-by-reversing-subarrays.c
bool canBeEqual(int* target, int targetSize, int* arr, int arrSize) {
    int count[1000] = {0};
    for (int i=0; i<targetSize; i++) {
        count[target[i]]++;
        count[arr[i]]--;
    }

    for (int j=0; j<1000; j++) {
        if (count[j]!=0) return false;
    }
    
    return true;
}

File: 1460-make-two-arrays-equal-by-reversing-subarrays/1460-make-two-arrays-equal-by-reversing-subarrays.py
class Solution(object):        
    def canBeEqual(self, target, arr):
        """
        :type target: List[int]
        :type arr: List[int]
        :rtype: bool
        """
        target_dict=dict()
        for t in target:
            if t in target_dict:
                target_dict[t]+=1
            else:
                target_dict[t]=1

        arr_dict = dict()
        for n in arr:
            if n in arr_dict:
                arr_dict[n]+=1
            else:
                arr_dict[n]=1
        
        return target_dict==arr_dict

        
